src/app/actions.ts

```diff
@@ -142,3 +142,54 @@ export async function getNearbySightings({
 
   return json;
 }
+
+/*
+ * NEARBY AND RECENT SIGHTINGS ACTIONS
+ */
+export async function getBirdImage(speciesName: string): Promise<{
+  query: {
+    pages: {
+      [key: string]: {
+        pageid: number;
+        ns: number;
+        title: string;
+        thumbnail: {
+          source: string;
+          width: number;
+          height: number;
+        };
+        pageprops: {
+          displaytitle: string;
+          defaultsort: string;
+        };
+      };
+    };
+  };
+}> {
```

Comment:
We can probably pull this inline type out into something named, like `BirdImageResponse` or similar. 

This would let us share it and use it with other functions that expect to make use of this functions return type.
```suggestion

export type BirdImageResponse = {
  query: {
    pages: {
      [key: string]: {
        pageid: number;
        ns: number;
        title: string;
        thumbnail: {
          source: string;
          width: number;
          height: number;
        };
        pageprops: {
          displaytitle: string;
          defaultsort: string;
        };
      };
    };
  };
}

export async function getBirdImage(speciesName: string): Promise<BirdImageResponse> {
```